src/main/thrift/parquet.thrift

```diff
@@ -190,6 +190,45 @@ enum FieldRepetitionType {
   /** The field is repeated and can contain 0 or more values */
   REPEATED = 2;
 }
+/**
+ * A structure for capturing metadata for estimating the unencoded, uncompressed size
+ * of data.
+ *
+ * Writers should populate all fields in this struct except for the exceptions listed per field.
+ */ 
+struct SizeEstimationStatistics {
+   /** 
+    * The number of logical physical bytes stored for BYTE_ARRAY data values. Logical bytes refers to the number
```

Comment:
"logical physical bytes" is a bit confusing

Reply:
yeah, tried rephrasing.